### PR TITLE
Add 3rd party description search

### PR DESCRIPTION
**Pull request by cookienc**
## Description

Signed-off-by: SEOKHO KIM [kim76744dev@gmail.com]
Issue #816 
Add description search on 3rd party list
- Add Description textarea
- Add query
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/resources/oss/fosslight/repository/PartnerMapper.xml [hunk 1]
```diff
@@ -225,6 +228,9 @@
 		<if test="!@oss.fosslight.util.StringUtil@isEmpty(modifier)">
 		   AND A.MODIFIER LIKE CONCAT('%',#{modifier},'%')
 		</if>
+		<if test="!@oss.fosslight.util.StringUtil@isEmpty(description)">
+			AND A.DESCRIPTION LIKE CONCAT('%',#{description},'%')
+		</if>
```
Comment (70825) [inline src/main/resources/oss/fosslight/repository/PartnerMapper.xml:233]: It looks better to modify it with the same indent 
```suggestion
		<if test="!@oss.fosslight.util.StringUtil@isEmpty(modifier)">
		   AND A.MODIFIER LIKE CONCAT('%',#{modifier},'%')
		</if>
		<if test="!@oss.fosslight.util.StringUtil@isEmpty(description)">
		   AND A.DESCRIPTION LIKE CONCAT('%',#{description},'%')
		</if>

```

**Review by 70825 — CHANGES_REQUESTED** [review]

File: src/main/resources/oss/fosslight/repository/PartnerMapper.xml (hunk 1 above)
Comment (cookienc) [reply src/main/resources/oss/fosslight/repository/PartnerMapper.xml:233]: Thanks for your review! I'll change it. 
Can I see any code convention files related to this project?

Comment (70825) [reply src/main/resources/oss/fosslight/repository/PartnerMapper.xml]: Someone else will be able to tell you the answer to your question.

Comment (Min-Kyungsun) [reply src/main/resources/oss/fosslight/repository/PartnerMapper.xml]: @cookienc 
The code convention has not set yet.
I will let you know when the code convention is set. 
Thanks. :)

**Comment by Min-Kyungsun** [comment]
Thank you for your contribution.
Please feel free to contribute FOSSLight Hub whenever you want.:)

**Pull request merged**